src/Commands/DiffCommand.php

```diff
@@ -61,7 +61,7 @@ private function printJobDiff($jobName)
         $jobDiff = $jobComparisonBusinessCase->getJobDiff($jobName);
 
         foreach ($jobDiff as $property => $diff) {
-            $diffLines = array_reverse(explode(PHP_EOL, $diff));
+            $diffLines = explode(PHP_EOL, $diff);
```

Comment:
It is. It should switch the line removing and adding a change, for example. It also should switch the opening and closing braces for a property.